Cargo.toml

```diff
@@ -84,3 +84,7 @@ debug = true
 
 [profile.bench]
 debug = true
+
+[patch.crates-io]
+zcash_note_encryption = { git = "https://github.com/daira/librustzcash.git", rev = "f07c4c4358f2d062e2ce9108069927bcf16117a8" }
```

Comment:
I checked that this is the correct intermediate commit (https://github.com/zcash/librustzcash/pull/633/commits/f07c4c4358f2d062e2ce9108069927bcf16117a8).

Reply:
This needs re-checking now that I've addressed your other comment.